samples/python/django/README.md

```diff
@@ -0,0 +1,82 @@
+|----------------------------------------|----------------------------------------------|
+| boolean                                | BoolField                                    |
+| bigint / int8                          | BigIntegerField                              |
+| varchar                                | CharField                                    |
+| text                                   | CharField                                    |
+| float8 / double precision              | FloatField                                   |
+| numeric(x, y)                          | DecimalField(max_digits=x, decimal_places=y) |
+| timestamptz / timestamp with time zone | DateTimeField                                |
+| bytea                                  | BinaryField                                  |
+| date                                   | DateField                                    |
+
+## Limitations
+The following limitations are currently known:
+
+| Limitation                           | Workaround                                                                                                                                         |
+|--------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| Schema updates                           | Cloud Spanner does not support the full PostgreSQL DDL dialect. Automated schema updates using `hibernate` are therefore not supported. It is recommended to set the option `hibernate.hbm2ddl.auto=none` (or `spring.jpa.hibernate.ddl-auto=none` if you are using Spring).               |
+| Generated primary keys               | Cloud Spanner does not support `sequences`. Auto-increment primary key is not supported. Remove auto increment annotation for primary key columns. The recommended type of primary key is a client side generated `UUID` stored as a string. |
+
+
+### Schema Updates
+Schema updates are not supported as Cloud Spanner does not support the full PostgreSQL DDL dialect. It is recommended to
+create the schema manually. Note that PGAdapter does support `create table if not exists` / `drop table if exists`.
+See [sample-schema.sql](src/main/resources/sample-schema-sql) for the data model for this example.
+
+### Generated Primary Keys
+`Sequences` are not supported. Hence, auto increment primary key is not supported and should be replaced with primary key definitions that
+are manually assigned. See https://cloud.google.com/spanner/docs/schema-design#primary-key-prevent-hotspots
+for more information on choosing a good primary key. This sample uses UUIDs that are generated by the client for primary
+keys.
+
+```java
+public class User {
+	// This uses auto generated UUID.
+    @Id
+    @Column(columnDefinition = "varchar", nullable = false)
+    @GeneratedValue
+    private UUID id;
+}
```

Comment:
We should add a 'Performance Considerations' section for Django as well. See https://github.com/GoogleCloudPlatform/pgadapter/blob/e7c4797ce35cd0fccc688a1269731762cd1c9c86/samples/python/sqlalchemy-sample/README.md#performance-considerations for an example.